ibis/backends/tests/test_asof_join.py

```diff
@@ -80,7 +80,9 @@ def time_keyed_right(time_keyed_df2):
 @pytest.mark.parametrize(
     ("direction", "op"), [("backward", operator.ge), ("forward", operator.le)]
 )
-@pytest.mark.notyet(["datafusion", "snowflake", "trino", "postgres", "mysql"])
+@pytest.mark.notyet(
+    ["datafusion", "snowflake", "trino", "postgres", "mysql", "bigquery"]
```

Comment:
I guess we can use correlated joins to implement asof join:
https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#correlated_join

This should be a follow-up, but deserves an issue to not forget it.

Reply:
I dunno. I think we should wait for someone to request it and not have a forever issue.

Reply:
We should try to avoid implementing complex features that no one has requested. At the same time, we also don't need an issue open for it that will never be closed. Someone can request it, and we can do the work then.